core/src/test/java/org/apache/gravitino/connector/TestCatalogOperations.java

```diff
@@ -198,17 +202,24 @@ public Table alterTable(NameIdentifier ident, TableChange... changes)
           throw new TableAlreadyExistsException("Table %s already exists", ident);
         }
       } else {
-        throw new IllegalArgumentException("Unsupported table change: " + change);
+        // do nothing
```

Comment:
why remove the error throw?

Reply:
Because we need to handle columnChanges in the below code, so we should not throw an exception.

Reply:
should change to:
```java
 else if (!(change instanceof TableChange.ColumnChange)){
        throw new IllegalArgumentException("Unsupported table change: " + change);
      }
```
because other table changes such as `UpdateComment` is not supported here

Reply:
can be updated when using